example/android/app/build.gradle

```diff
@@ -1,5 +1,6 @@
 apply plugin: "com.android.application"
 apply plugin: "com.facebook.react"
+apply plugin: "kotlin-android"
```

Comment:
Here, I just added kotlin support for example android app, to make it possible to include kotlin file with the example plugin